templates/includes/beta-banner.html

```diff
@@ -1,10 +1,15 @@
+{% load wagtailcore_tags %}
 <div class="beta-banner" aria-label="About The National Archives' beta">
-    <p class="beta-banner__standfirst">What does the 'beta' label mean?</p>
-    <p class="beta-banner__text">
-        This beta site contains new services and features currently in development. Many of these features are works in 
-        progress and are being updated regularly.  You can help us improve them by providing feedback as you use the site.
-    </p>
-    <div class="beta-banner__options">
-        <a class="beta-banner__link" href="/">Find out more</a>
-    </div>
+
+         <div class="beta-banner__standfirst">
+         {{ settings.core.SiteSettings.beta_banner_standfirst }}
+         </div>
+        <div class="beta-banner__text">
+            {{ settings.core.SiteSettings.beta_banner_text|richtext }}
+        </div>
+        <div class="beta-banner__options">
+            <a class="beta-banner__link" href="{{ settings.core.SiteSettings.beta_banner_link }}">
```

Comment:
@JamesWChan Could the `rel="noopener noreferrer"` be added with JS? That feels like more of a front-end concern than a backend one. This should be easy to detect with something like:

```
const isExternalURL = (url) => new URL(url).origin !== location.origin;
```

Reply:
@ababic - it can’t I’m afraid. As the attribute would be part of the core experience (i.e. not an enhancement), we can’t rely on JS to deliver it.

Reply:
But `noreferrer` is an SEO nicety thing, right? It's not important to the core user experience (which is what progressive enhancement is all about).  And `noopener` carries some security benefits, but is probably unnecessary in this case, as the target will always be trusted.

Reply:
As per Section 9 of the GDS Service Manual:

“Service teams should actively identify security and privacy threats to the service, and have a robust, proportionate approach to securing information and managing fraud risks.”

Linking to a third party is a potential security threat. Yes, the third party we link to would most likely be fine, which is why I'm suggesting a proportionate approach (i.e. adding just a small bit of code which you've said should be simple to implement).

This would mean that we'd be doing everything we can to make the core experience of the highest quality (which is also what progressive enhancement is all about).

Source - https://www.gov.uk/service-manual/service-standard/point-9-create-a-secure-service

Reply:
I'm keen to identify the _right_ approach here @JamesWChan. Is this only a problem with the banner link, or a wider issue? Is a backend solution capable of solving this problem everywhere it needs to be solved? I'm all for writing code, I've just seen enough 'temporary' solutions on this project to last me a life time.